libc/cmake/modules/LLVMLibCCompileOptionRules.cmake

```diff
@@ -241,17 +241,18 @@ function(_get_common_test_compile_options output_var c_test flags)
       list(APPEND compile_options "-ffixed-point")
     endif()
 
-    # list(APPEND compile_options "-Wall")
-    # list(APPEND compile_options "-Wextra")
+    list(APPEND compile_options "-Wall")
+    list(APPEND compile_options "-Wextra")
     # -DLIBC_WNO_ERROR=ON if you can't build cleanly with -Werror.
     if(NOT LIBC_WNO_ERROR)
-      # list(APPEND compile_options "-Werror")
+      list(APPEND compile_options "-Werror")
+      list(APPEND compile_options "-Wno-global-constructors")
     endif()
```

Comment:
Let's keep all of the additions together, then have this condition of Werror at the end?

Reply:
Let me know if this still needs to be moved?